site/en/docs/devtools/javascript/sources/index.md

```diff
@@ -28,25 +29,38 @@ Use the Chrome DevTools **Sources** panel to:
 
 Use the **Page** pane to view all of the resources that the page has loaded.
 
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/smmuE51BssHZBgSb2Ytb.png", alt="The Page pane", width="800", height="536" %}
-
-**Figure 1**. The **Page** pane
+{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/smmuE51BssHZBgSb2Ytb.png", alt="The Page pane.", width="800", height="536" %}
 
 How the **Page** pane is organized:
 
-- The top-level, such as `top` in **Figure 1**, represents an [HTML frame][6]. You'll find `top` on
+- The top-level, such as `top` on the screenshot above, represents an [HTML frame][6]. You'll find `top` on
   every page that you visit. `top` represents the main document frame.
-- The second-level, such as `developers.google.com` in **Figure 1**, represents an [origin][7].
+- The second-level, such as `developers.google.com` on the screenshot above, represents an [origin][7].
 - The third-level, fourth-level, and so on, represent directories and resources that were loaded
-  from that origin. For example, in **Figure 1** the full path to the resource
+  from that origin. For example, on the screenshot above, the full path to the resource 
   `devsite-googler-button` is `developers.google.com/_static/19aa27122b/css/devsite-googler-button`
 
 Click a file in the **Page** pane to view its contents in the **Editor** pane. You can view any type
 of file. For images, you see a preview of the image.
 
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/ByErR7HFeQ7zOVjF4Wyp.png", alt="Viewing a file in the Editor pane", width="800", height="530" %}
+{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/ByErR7HFeQ7zOVjF4Wyp.png", alt="Viewing a file in the Editor pane.", width="800", height="530" %}
+
+### Group authored and deployed files on the Page pane {: #group-authored-and-deployed }
+
+{% Aside %}
+**Note**: This is an {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/XfSWf04g2cwpnFcmp40m.svg", alt="Experimental.", width="20", height="20" %} experimental feature available in Chrome from version 104.
+{% endAside %}
+
+When developing web applications using frameworks (for example, [React](https://reactjs.org/), [Angular](https://angular.io/)), it can be difficult to navigate sources due to the minified files generated by the build tools (for example, [Webpack](https://webpack.js.org/), [Vite](https://vitejs.dev/)).
```

Comment:
```suggestion
When developing web applications using frameworks (for example, [React](https://reactjs.org/) or [Angular](https://angular.io/)), it can be difficult to navigate sources due to the minified files generated by the build tools (for example, [Webpack](https://webpack.js.org/) or [Vite](https://vitejs.dev/)).
```